docs/getting-started/dockstore-workflows.rst

```diff
@@ -47,149 +48,125 @@ below:
+files and other data.
 
-Now that you have linked your credentials and refreshed, there should be
-one stub per repo that you are the owner of or have rights to see in
-GitHub.
+To register content on Dockstore, you must have an account on Dockstore and
+link the necessary third-party accounts. Once this is done you can register
+workflows from the My Workflows page.
 
 Quick Registration via the Web UI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Quick registration is best used for workflows that follow the simple format
+that Dockstore suggests. It can still be used if your workflows are
+non-standard format, however there can be some drawbacks.
 
-In the authenticated Web UI, navigate to 'My Workflows' to begin
-managing workflows imported through your linked account(s). These pages
-will allow you to quickly register workflows that follow a particularly
-simple format (look below to manual registration for more complex
-formats). For quick registration, we look through your GitHub, Bitbucket
-and GitLab accounts and create a stub for each one. You can refresh and
-publish each repo that you identify as a real workflow in order to get,
-edit, and display additional information such as available versions.
-
-In the above image, the left side menu is a list of all workflow
-repositories associated with the user, grouped lexicographically by
-namespace. Words encapsulated in parentheses denotes a custom toolname
-if provided. Detailed information and links for each Workflow are
-located on the 'Info' tab. Settings, such as the path to the CWL/WDL
-descriptor, can be modified on a per-tag basis in the 'Versions' tab.
-The Descriptors and test parameter files may be viewed in the 'Files'
-tab, by the Version tag (corresponding to a Git tag/branch). Finally,
-'Manage labels' (located above the tabs) allows you to add/edit keywords
-that you want to be associated with a workflow for efficient searching
-and grouping.
-
-Just like with Tools, a workflow is not visible on the public
-'Workflows' listing unless it is published. To publish a container,
-click the blue 'Publish' button in the top-right corner.
+Some users have multiple workflows within one Git repository, however each
+workflow entry on Dockstore only contains a single workflow. This is
+a problem as the Git path is used to uniquely identify a Dockstore workflow.
+The solution to this is to allow users to specify a workflow name that is
+appended to the Dockstore path. This would allow them to have multiple
+Dockstore workflows with the same Git repository. Quick registration does
+not allow you to create workflows with workflow names.
+To do that you must do manual registration, which is described later.
 
-Manual Registration of Workflows
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Refresh All
+^^^^^^^^^^^
+Refresh all will look at all of your third-party accounts and do the following:
+
+- Create stub workflows for all Git repositories which do not exist on
```

Comment:
Same comment/question regarding stubs

Reply:
Replied to this in a general comment below.